packages/node_modules/@webex/internal-plugin-dss/src/dss.ts

```diff
@@ -194,14 +194,17 @@ const DSS = WebexPlugin.extend({
    * Retrieves basic information about a list entities within an organization
    * @param {Object} options
    * @param {UUID} options.ids the id of the entity to lookup
+   * @param {UUID} options.entityProviderType the provider to query (optional)
    * @returns {Promise} Resolves with an array of entities found
    */
   lookup(options: LookupOptions) {
-    const {ids} = options;
+    const {ids, entityProviderType} = options;
+
+    const resource = entityProviderType ? `/lookup/orgid/${this.webex.internal.device.orgId}/entityprovidertype/${entityProviderType}` : `/lookup/orgid/${this.webex.internal.device.orgId}/identities`;
 
     return this._request({
       dataPath: 'lookupResult.entities',
-      resource: `/lookup/orgid/${this.webex.internal.device.orgId}/identities`,
+      resource,
```

Comment:
nitpick: Could save a small amount of repetition if we did something like:

```ts
const resource = entityProviderType ? `entityprovidertype/${entityProviderType}` : 'identities';

return this._request({
  ...
  resource: `/lookup/orgid/${this.webex.internal.device.orgId}/${resouce}`,
  ...
});
```

Functionally makes no difference of course.